libr/util/str.c

```diff
@@ -216,7 +216,7 @@ R_API int r_str_binstr2bin(const char *str, ut8 *out, int outlen) {
 // Returns the permissions as in integer given an input in the form of rwx, rx,
 // etc.
 R_API int r_str_rwx(const char *str) {
-	int ret = atoi (str);
```

Comment:
Also, this would allow to keep this "magic numbers" as our internal representation of permission and if one day we are going to change that for whatever reason, nobody that uses the commands with "rwxm" will be affected